app/controllers/integrated/food_assistance_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Integrated
+  class FoodAssistanceController < FormsController
+    def self.skip?(current_application)
+      if current_application.members.count == 1 || current_application.unstable_housing?
+        current_application.members.each { |member| member.update(requesting_food: "yes") }
```

Comment:
I feel a little hesitant to do database updates in the skip? method since it's a class method called during the course of navigation, rather than once we visit the controller. I think it should only get called when we're trying to figure out if we should visit that page, but I think that we can't guarantee that's true with the existing interface. but, it's also nice to not have to tackle this update in an unrelated controller. would love to talk about this when we implement the next time!